app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,94 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  # create ServiceTemplate and supporting ServiceResources and ResourceActions
+  # options
+  #   :name
+  #   :description
+  #   :service_template_catalog_id
+  #   :config_info
+  #     :provision
+  #       :dialog_id (or)
+  #       :new_dialog_name
```

Comment:
For this initial work we will not support `new_dialog_name`. User needs to prepare a dialog in advance.